library_base/src/main/java/com/igorwojda/showcase/base/presentation/extension/ContextExtensions.kt

```diff
@@ -20,3 +22,5 @@ inline fun <reified T : AppCompatActivity> Context.startActivity(
     intent.putExtras(bundleOf(*extras))
     startActivity(intent)
 }
+
+fun Context.dimension(@DimenRes dimenRes: Int) = resources.getDimension(dimenRes)
```

Comment:
I don't think this is the correct name. 

This is function that defines an action and I prefer to follow `doSomething` convention (eg. `clearCache`, `getUser`, `setData`, `removeItem`...). My preference comes mainly from the fact that in Kotlin (as opposed to Java) we also have properties that describe the sate (`name`, `size`, `dimension`).

Please take a look at this
https://blog.kotlin-academy.com/kotlin-should-i-define-function-or-property-6786951da909